stringcalculator/stringcalculator/Calculator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace stringcalculator
+{
+    public class Calculator
+    {
+        public int Add(string v)
+        {
+            int result = 0;
+            if (v == "")
```

Comment:
We don't need this yet - let's remove it for now and just have the method `return 0;`